Tabloid/client/src/components/CategoryList.js

```diff
@@ -25,8 +25,8 @@ const CategoryList = () => {
       <div className="container">
         <div className=" mt-4 row justify-content-center">
           {categories.map((category) => (
-            <ListGroupItem><Category category={category} key={category.id} />
-            <Link to={`/categories/edit/${category.id}`}>Edit</Link>{" "}
+            <ListGroupItem key={`editlink--${category.id}`}><Category category={category} key={category.id} />
+            <Link to={`/categories/edit/${category.id}`} key={`editlink--${category.id}`}>Edit</Link>{" "}
```

Comment:
Do you get a key error when this Link component doesn't have a key attribute?

Reply:
It appears I only get the error when it's not on the ListGroupItem, so I can probably just use it there.

Reply:
Set key to ListGroupItem Only.